### PR TITLE
New Script: Quit all Airdrop processes

### PR DESCRIPTION
**Pull request by linus569**
## Description

The script kills all AirDrop processes running on the Mac. I have the problem that AirDrop freezes from time to time and maybe I'm not the only one. This script makes it easy to quit all processes and use AirDrop again.

## Type of change

- [x] New script command

## Screenshot
ezgif-1-d74516c215be

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/system/quit-airdrop-process.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Quit AirDrop
```
Comment (unnamedd) [inline commands/system/quit-airdrop-process.sh:5]:
```suggestion
# @raycast.title Kill AirDrop Processes
```

File: commands/system/quit-airdrop-process.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+# @raycast.mode silent
+# @raycast.packageName System
+
+# Optional parameters:
+# @raycast.icon 🔫
+
+# @Documentation:
+# @raycast.description Quits all AirDrop processes
```
Comment (unnamedd) [inline commands/system/quit-airdrop-process.sh:13]:
```suggestion
# @raycast.description Force kill all AirDrop processes
```

File: commands/system/quit-airdrop-process.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```
Comment (unnamedd) [inline commands/system/quit-airdrop-process.sh:1]: Will fit better if you rename this file to `kill-airdrop.processes.sh`.

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hi @linus569, I made some small comments to keep the consistency in your Script Command.

Thanks for your first contribution and welcome aboard!

**Comment by linus569** [comment]
Thanks for your help, improved it with your suggestions

**Review by RSO — APPROVED** [review]

**Comment by RSO** [comment]
Looks good to me, thanks!

**Pull request merged**